packages/ui/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "private": true,
+  "name": "@aws-amplify/ui",
+  "version": "0.0.1-next.0",
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
+  "exports": {
+    ".": "./dist/index.mjs",
+    "./styles.css": "./dist/styles.css",
+    "./theme.css": "./dist/theme.css"
+  },
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "prebuild": "rimraf dist",
+    "build": "run-s build:sd build:css build:js --print-label",
+    "build:css": "./scripts/generateCSS.js",
+    "build:js": "tsup --minify",
+    "build:sd": "style-dictionary build --config ./sd.config.js",
```

Comment:
One idea: do we want to consider `"build:theme-base": run-s build:sd build:css` so that developers working on just the theme can use this one command to build?

Reply:
I think the best solution for that is to have the CSS as an output of 1 build process – otherwise there's no watch-mode either =/

It's in the PR description, but I de-scoped it since it would change our SASS usage to use PostCSS.

But in my mind, we'd do `yarn ui dev` and if you're working on CSS, it would watch & output that on change.

The only unknown I have is a workflow for watching the style-dictionary theme stuff 🤔

Reply:
Got it, that makes sense to me.